### ISSUE TITLE
Compile-Time Dependency on 'org.codehaus.groovy:groovy-all:2.3.6'

### ISSUE THREAD

**Issue by stellingsimon** [ical4j/ical4j-connector#31]
We're experiencing problems in our build because ical4j-connector pulls in a compile-time dependency on `groovy-all-2.3.6`, which is fairly old and incompatible with `groovy-all:2.4`:

```
Caused by: groovy.lang.GroovyRuntimeException: Conflicting module versions. Module [groovy-all is loaded in version 2.4.6 and you are trying to load version 2.3.6
```

A quick search in the ical4j-connector repo didn't reveal an obvious reason why this dependency is needed as a compile-time dependency, but I guess I'm missing something. Or could it be dropped or made a test-only/compile-only dependency instead?

**Comment by benfortuna** [comment]
Hi Simon, yes i can definitely make it optional dependency at least, will confirm shortly

**Comment by benfortuna** [comment]
So I removed the groovy dependency completely, I don't think it was required anymore. I've just released 1.0.0 which includes this change, let me know of any issues.
